README.md

```diff
@@ -77,8 +77,10 @@ To build and run the benchmarks, please install the necessary dependencies. Plea
 3. `yarn install`
 4. `yarn android --mode release` to build the app with optimisations.
 5. `yarn android` to build the app in debug mode.
-</details>
 
+> Note: For Android you will may be able to switch architectures during run time. To switch to the old architecture, set the newArchEnabled variable to false in the [gradle.properties](./App/android/gradle.properties) file.
```

Comment:
This is a bit confusing as you talk about runtime in the first sentence, but the sencond sentence is about changing a gradle property (build time).

Reply:
Sorry I realise there is a typo: Meant to say "For Android you may not be able to switch architectures during run time"